docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,649 @@
+# Week 1: Project Setup and Terminal Basics
+
+## Agenda
+
+- Project Overview
+- Project Setup
+    - Installing Required Software
+    - Clone the repositories
+- Terminal Basics
+    - Common Commands
+    - Build Tools
+    
+## Basic Plan
+
+- Jumpstart Overview
+    - What is Jumpstart?
+    - Who is this program for?
+    - How should I prepare?
+    - What am I going to be building?
+        - End Goal
+    - What technologies will I be learning?
+    - How are the projects organized?
+        - Common Files
+        - Frontend
+        - Backend
+    - What will each week look like?
+    - Important Links
+        - Making edits to the dictionary
+- Project Setup
+    - Installing IntelliJ
+    - Cloning the repositories with Git
+    - Installing required apps
+- Terminal Basics
+    - Common Commands 
+        - cd and filepaths
+        - ls
+        - pwd
+        - mv
+        - cp
+        - mkdir
+        - rm
+        - cat
+        - vim
+        - nano
+        - sudo
+        - man
+    - Build Tools
+        - npm
+        - Maven
+        
+## Jumpstart Overview
+
+First, we'll be going over the general plan we have for Jumpstart over the next few months.
+
+### What is Jumpstart?
+
+From [our website](https://c4cneu.com/jumpstart).
+
+>Jumpstart is Code4Community's new program that focuses on teaching the basics of web 
+>development to beginners. 
+
+Over the next couple of weeks (we have 9-10 sessions planned including this one), we'll 
+be going through a lot of the fundamentals you'll need to know for developing an interactive 
+web application. Our aim is to help you understand and have the tools for developing a
+personal project of your own in the future.
+
+### Who is this program for?
+
+This program is honestly for anyone interested in software development or interested in the
+inner workings of a basic web application. Non-computer science majors are welcome! While a 
+background in computer programming is recommended, as long as you complete the recommended 
+preparation down below, you'll probably be well prepared for this course. We'll also be here 
+with dedicated tutors monitoring a special Slack channel and weekly office hours if you have
+any questions, no matter how silly they may seem.
```

Comment:
instead of "no matter how silly they may seem" it may be better to say "because there is no stupid question"

Reply:
Got it! I also bolded it.